disnake/interactions/base.py

```diff
@@ -622,9 +622,12 @@ async def defer(self, *, ephemeral: bool = False) -> None:
 
         Parameters
         -----------
+        show_message: :class:`bool`
+            Indicates whether or not the respond will be a message. Shows a loading screen to the user.
+            This only applies for interactions of type :attr:`InteractionType.component`.
```

Comment:
```suggestion
        show_message: :class:`bool`
            Indicates whether the response will be a message with thinking state (bot is thinking...)
            This only applies for interactions of type :attr:`InteractionType.component`.
```
Basically to follow the documentation of `Interaction.original_message`.